### PR TITLE
Debt/met 4765 dcmi period date extractor cleanup

### PR DESCRIPTION
**Pull request by stzanakis**

### CODE REVIEW

**Review by jeortizquan — COMMENTED** [review]
I think this one is also good, have a look on that edge case.

File: metis-normalization/src/main/java/eu/europeana/normalization/dates/extraction/dateextractors/DcmiPeriodDateExtractor.java [hunk 1]
```diff
@@ -6,115 +6,129 @@
...
+  private static final String NON_SPACE_NON_SEMICOLON = "[^\\s;]*";
+  private static final String NON_SPACE_NON_LINE_END = "[^\\s$]*";
+  private static final String VALUE_ENDING = "(?:;|$)";
+  private static final String SPACE_VALUE_ENDING = "\\s*" + VALUE_ENDING;
+  private static final String EQUALS_SPACES_WRAPPED = "\\s*=\\s*";
+  private static final String DCMI_FIELD_REGEX =
+      EQUALS_SPACES_WRAPPED + "(" + NON_SPACE_NON_SEMICOLON + "|" + NON_SPACE_NON_LINE_END + ")" + SPACE_VALUE_ENDING;
+  private static final Pattern DCMI_PERIOD_SCHEME_PATTERN = Pattern.compile("scheme" + DCMI_FIELD_REGEX);
+  private static final Pattern DCMI_PERIOD_START_PATTERN = Pattern.compile("start" + DCMI_FIELD_REGEX);
+  private static final Pattern DCMI_PERIOD_END_PATTERN = Pattern.compile("end" + DCMI_FIELD_REGEX);
```
[the remark this answers is not in the document]
Comment (stzanakis) [reply metis-normalization/src/main/java/eu/europeana/normalization/dates/extraction/dateextractors/DcmiPeriodDateExtractor.java:34]: Do I need to check something here or?

File: metis-normalization/src/test/java/eu/europeana/normalization/dates/extraction/dateextractors/DcmiPeriodDateExtractorTest.java [hunk 2]
```diff
@@ -20,84 +18,102 @@
...
+            null, "1929", "1939"),
+
+        //Name at the beginning without field name and spaces at wrapped
+        Arguments.of("   The Great Depression   ; start=1929; end=1939;",
+            null, "1929", "1939"),
+
+        //Normal case
+        Arguments.of("name=The Great Depression; start=1929; end=1939;",
+            "The Great Depression", "1929", "1939")
+    );
```
[the remark this answers is not in the document]
Comment (stzanakis) [reply metis-normalization/src/test/java/eu/europeana/normalization/dates/extraction/dateextractors/DcmiPeriodDateExtractorTest.java:117]: No we don't support that.

File: metis-normalization/src/main/java/eu/europeana/normalization/dates/extraction/dateextractors/DcmiPeriodDateExtractor.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (jeortizquan) [reply metis-normalization/src/main/java/eu/europeana/normalization/dates/extraction/dateextractors/DcmiPeriodDateExtractor.java:34]: no. we are good, is just documentation.

**Pull request merged**